pbr/files/etc/init.d/pbr.init

```diff
@@ -1632,6 +1632,8 @@ interface_routing() {
 					fi
 					# shellcheck disable=SC2086
 					while read -r i; do
+						# shellcheck disable=SC2001
+						i="$(echo "$i" | sed 's/linkdown$//')"
```

Comment:
```suggestion
						i="$(echo "$i" | sed 's/ linkdown$//')"
						i="$(echo "$i" | sed 's/ onlink$//')"
```

Reply:
Again, thank you for bringing it up. After digging a bit into this (with the example [here](https://unix.stackexchange.com/questions/398810/default-route-with-linkdown)), I wonder if this would be a more robust solution. Since you said you've looked into this, please comment on this suggestion.

Also, could you please advise if the same needs to be added to the IPv6 routes processing? 

Anyways, please also bump the revision number in the Makefile.

Reply:
Looks good. And I can see you've released this in 762070b, and it's working.

Thanks very much